### PR TITLE
Refactored delivery queue code to fit new POs

### PR DESCRIPTION
**Pull request by jbolles94**

### CODE REVIEW

File: src/receiving_queue/tables/ReceivingQueueTable.jsx [hunk 1]
```diff
@@ -100,15 +101,34 @@ const ReceivingQueueTable = ({
...
+            data?.queue.workOrderPOQueue.forEach((e) => {
+              queueTableData.push({
+                id: e._id,
+                manifest: e.po,
+                source: e.source,
+                label: e.label,
+                poType: WORK_ORDER_PO,
+              });
+            });
+            // data?.queue?.incomingDeliveries.forEach((e) => {
```
Comment (joshplumley) [inline src/receiving_queue/tables/ReceivingQueueTable.jsx:123]: Remove

**Comment by joshplumley** [comment]
WARNING in src/receiving_queue/tables/ReceivingQueueTable.jsx
  Line 16:10:  'WORK_ORDER_PO' is defined but never used  no-unused-vars
  Line 16:25:  'CONSUMABLE_PO' is defined but never used  no-unused-vars

**Comment by joshplumley** [comment]
Clicking on a row within the receiving queue dropdown produces this error:

GridRow.js:189 Uncaught TypeError: Cannot read properties of undefined (reading 'type')
    at GridRow.js:189:1
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:1)
    at invokeGuardedCallback (react-dom.development.js:4056:1)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4070:1)
    at executeDispatch (react-dom.development.js:8243:1)
    at processDispatchQueueItemsInOrder (react-dom.development.js:8275:1)
    at processDispatchQueue (react-dom.development.js:8288:1)
    at dispatchEventsForPlugins (react-dom.development.js:8299:1)
    at react-dom.development.js:8508:1

File: src/receiving_queue/tables/ReceivingQueueTable.jsx (hunk 1 above)
Comment (jbolles94) [reply src/receiving_queue/tables/ReceivingQueueTable.jsx]: Removed.

**Comment by jbolles94** [comment]
@joshplumley I don't get any of these anymore if you want to take another look.

**Comment by joshplumley** [comment]
> @joshplumley I don't get any of these anymore if you want to take another look.

Looking good

**Review by joshplumley — APPROVED** [review]

**Pull request merged**